test/fixtures/module-require-symlink/symlinked.js

```diff
@@ -6,8 +6,8 @@ const path = require('path');
 const linkScriptTarget = path.join(common.fixturesDir,
   '/module-require-symlink/symlinked.js');
```

Comment:
Now that you are changing this file, I would prefer to write this as
```js
path.join(common.fixturesDir, 'module-require-symlink', 'symlinked.js')
```
Can you please do this as well?